src/lib/PostListCard.svelte

```diff
@@ -12,6 +23,25 @@
     day: 'numeric',
     year: 'numeric',
   })
+
+  export const FaIcons = {
```

Comment:
You're right. No clue why I put `export` since I don't use it elsewhere aside from in PostCards and PostPage so I will update that - however, there were other issues I ran into while doing this on a per element basis. Each element is slightly different and so just entering this into each element seemed easiest for me. I would love to centralize and generalize it but I wasn't quite sure the best way to do it based on the above reason.